radis/io/hitran.py

```diff
@@ -1437,6 +1488,14 @@ def fetch_hitran(
         output=output,
     )
 
+    #creating consolidated .hdf5 file for extra_params of all isotopes of molecule 
+    if(len(extra_params) != 0):
+        import vaex
+        vaex_df = vaex.from_pandas(temp_df, copy_index=False)
+        vaex_df.export_hdf5(join(local_databases, molecule+"extra_params.hdf5"))
+        
```

Comment:
this is not the right place to create a file. Can't you make the file creation happen in `download_and_parse` ? 

Note 2 : line 1483 :  you'll probably need to ask to load extra columns , i.e. add the required parameters ("gamma_CO2", "n_CO2" )  to `columns`

Reply:
In today's commit, I have created the consolidated `.hdf5` file for `extra_params` of all isotopes of the molecule  in `download_all_isotopes` since we are downloading extra_params for all isotopes.

Also,  rather than loading  required parameters in `columns` 
I've added a new parameter in `extra_params` in `fetch_hitran()` method  which I also had to add to `class DatabaseManager` in `dbmanager.py`.
So, through `extra_params` a user can pass the broadening coefficients like 'gamma_CO2, n_CO2' in the `fetch_hitran() `method and a `.HDF5` file for those attributes will be created in the same location.
For instance,
if user is calling -
````
from radis.io.hitran import fetch_hitran
fetch_hitran("CO",["gamma_CO2","n_CO2"])
````
then a file `CO.hdf5 `will be created with the basic attributes and a file `CO_gamma_CO2_n_CO2.hdf5` will be created that has only `gamma_CO2` and `n_CO2` values for CO.